src/tree/tree.scss

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../icon/index';
+
+$iui-active-outline: thin solid t(iui-color-foreground-primary);
+
+@mixin iui-tree {
+  @include themed {
+    background-color: t(iui-color-background-1);
+  }
+  list-style: none;
+  padding-left: 0;
+}
+
+@mixin iui-tree-node {
+  @include iui-focus;
+  display: flex;
+  cursor: pointer;
+
+  > .iui-checkbox {
+    margin-left: $iui-s;
+    margin-right: $iui-s;
+  }
+
+  > .iui-node-content {
+    display: flex;
+    align-items: center;
+    padding: $iui-xxs;
+    box-sizing: border-box;
+    min-height: $iui-baseline * 3;
+    margin-left: calc(36px * (var(--level, 0)));
+    overflow: hidden;
+
+    > .iui-icon {
```

Comment:
Have you checked test images?
![image](https://user-images.githubusercontent.com/36186912/143397742-4787e01c-8775-497c-9784-42837f8f802e.png)
You might need to add `flex-shrink: 0`

Reply:
Added